src/net/sourceforge/kolmafia/AreaCombatData.java

```diff
@@ -1178,454 +1178,521 @@ public String getZone() {
+          if (monster.equals("huge ghuol")) {
+            return evilness > 0 && evilness <= 25 ? 1 : 0;
+          } else if (monster.equals("gaunt ghuol") || monster.equals("gluttonous ghuol")) {
+            return evilness > 25 ? 1 : 0;
+          }
+          break;
+        }
+      case "The Defiled Niche":
+        {
+          int evilness = Preferences.getInteger("cyrptNicheEvilness");
+          if (monster.equals("gargantulihc")) {
+            return evilness > 0 && evilness <= 25 ? 1 : 0;
+          } else {
+            return evilness > 25 ? 1 : 0;
+          }
+        }
+      case "The Defiled Nook":
+        {
+          int evilness = Preferences.getInteger("cyrptNookEvilness");
+          if (monster.equals("giant skeelton")) {
+            return evilness > 0 && evilness <= 25 ? 1 : 0;
+          } else {
+            return evilness > 25 ? 1 : 0;
+          }
+        }
+      case "Haert of the Cyrpt":
+        if (monster.equals("Bonerdagon")
+            && QuestDatabase.isQuestLaterThan(Quest.CYRPT, QuestDatabase.STARTED)) {
+          return 0;
+        }
+        break;
+      case "The F'c'le":
+        if (monster.equals("clingy pirate (female)")) {
+          return KoLCharacter.getGender() == KoLCharacter.MALE ? 1 : 0;
+        } else if (monster.equals("clingy pirate (male)")) {
+          return KoLCharacter.getGender() == KoLCharacter.FEMALE ? 1 : 0;
+        }
+        break;
+      case "Summoning Chamber":
+        if (monster.equals("Lord Spookyraven") && QuestDatabase.isQuestFinished(Quest.MANOR)) {
+          return 0;
+        }
+        break;
+      case "An Overgrown Shrine (Northwest)":
+        // Assume lianas are dealt with once Apartment opened. Player may leave without doing so,
+        // but
+        // that's
+        // abit niche for me to care!
+        if (monster.equals("dense liana")
+            && Preferences.getInteger("hiddenApartmentProgress") > 0) {
+          return 0;
+        }
+        break;
+      case "An Overgrown Shrine (Northeast)":
+        // Assume lianas are dealt with once Office opened. Player may leave without doing so, but
+        // that's
+        // abit niche for me to care!
+        if (monster.equals("dense liana") && Preferences.getInteger("hiddenOfficeProgress") > 0) {
+          return 0;
+        }
+        break;
+      case "An Overgrown Shrine (Southwest)":
+        // Assume lianas are dealt with once Hospital opened. Player may leave without doing so, but
+        // that's
+        // abit niche for me to care!
+        if (monster.equals("dense liana") && Preferences.getInteger("hiddenHospitalProgress") > 0) {
+          return 0;
+        }
+        break;
+      case "An Overgrown Shrine (Southeast)":
+        // Assume lianas are dealt with once Bowling Alley opened. Player may leave without doing
+        // so,
+        // but that's
+        // abit niche for me to care!
```

Comment:
spotless has really knackered this file's comments.

Can you go through and combine them then reformat? Or make spotless not do this?